matRad_calcParticleDose.m

```diff
@@ -161,6 +150,22 @@
     rot_coordsV(:,2) = rot_coordsV(:,2)-stf(i).sourcePoint_bev(2);
     rot_coordsV(:,3) = rot_coordsV(:,3)-stf(i).sourcePoint_bev(3);
     
+    % Calcualte radiological depth cube
+    lateralCutoff = 50;
+    fprintf(['matRad: Calculating radiological depth cube for beam ' num2str(i) ' ...']);
```

Comment:
will this printf command result in messed up command line output? with regard to matRad_progress.

I agree that we should state that we are doing the ray tracing but maybe have percent per beam form now on.

also for photons!